ets-demo/etsdemo/tests/test_main.py

```diff
@@ -45,8 +46,18 @@ class TestMain(unittest.TestCase):
     @require_gui
     def test_main(self):
         # Main function must be launchable even if there are no data available.
-        with mock_iter_entry_points({}), mock_demo_launch():
+        argv = ["etsdemo", "-v"]
+        mocked_io = io.StringIO()
+        with mock_iter_entry_points({}), \
+                mock_demo_launch(), \
+                mocked_io, \
+                mock.patch("sys.stdout", mocked_io), \
+                mock.patch("sys.stderr", mocked_io), \
+                mock.patch("sys.argv", argv):
             main_module.main()
+            log_content = mocked_io.getvalue()
+
+        self.assertIn("Found 0 resource(s).", log_content)
```

Comment:
I think I need to add a test instead of modifying an existing one. One test for the normal case where we have no logging. The other one with some logging info.

Reply:
Done.